activestorage/test/models/attached/many_test.rb

```diff
@@ -593,4 +593,16 @@ def highlights
       User.remove_method :highlights
     end
   end
+
+  test "loading associations after attaching" do
+    funky = create_blob(filename: "funky.jpg")
+    town = create_blob(filename: "town.jpg")
+
+    @user.highlights.attach(funky)
+    @user.reload
+    @user.highlights.attach(town)
+
+    assert_not_empty @user.highlights_attachments
+    assert_equal @user.highlights_blobs.count, 2
+  end
```

Comment:
This and the test directly below are part of the separate commit mentioned above. 

Happy to modify as needed, but I thought it would be good to have a test to ensure that the associations get loaded when creating an attachment. I suppose this would be the default expected behavior for a model field, but attachments work differently from regular model fields, so good to have confirmation here.